salttesting/case.py

```diff
@@ -330,22 +333,26 @@ def detach_from_parent_group():
                     pass
             # pylint: enable=maybe-no-member
 
-        data = process.communicate()
-        process.stdout.close()
+        # TODO Remove this?
+        process.communicate()
```

Comment:
If `communicate()` brings no additional problems leave it. I _think_ it does more that handle stderr and stdout behind the scenes. I could be wrong.